### PR TITLE
gives the BSO a hardsuit Fucking Finally

### PR DESCRIPTION
**Pull request by starch70**
## About the PR
i gave the BSO a hardsuit finally and also added a suit storage locker for it

(thanks perzonaz for the hardsuit sprites its fire)

## Why / Balance
(https://discord.com/channels/1202734573247795300/1296989133805195284)

## Technical details
adds suit_storage.yml for the fucking uhh suit storage locker
edits the fills for the blueshield officer locker
(not making a separate hardsuit version because then on the maps without it it would be impossible to get it because the locker teleporter would give you the version without locker and im really too lazy to figure out where the yaml/shitcode for the teleporter is)

## Media
bso

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl:
- add: added BSO hardsuit (finally) (stop stealing sec/engi/anything else now losers)

## Summary by CodeRabbit

- **New Features**
	- Introduced new locker fill entities, enhancing storage options.
	- Added new hardsuit helmets with distinct attributes for enhanced gameplay.
	- Introduced new hardsuit entities with varying protection and resistance capabilities.
	- Added metadata files for graphical assets to manage visual states effectively.

- **Bug Fixes**
	- Modified existing entities to improve clarity and functionality. 

- **Documentation**
	- New JSON metadata files created for better asset management and licensing information.

### CODE REVIEW

**Comment by whateverusername0** [comment]
test fail:

**Comment by starch70** [comment]
> test fail:

i dont know why its doing that :godo:

**Comment by starch70** [comment]
> > test fail:
> 
> i dont know why its doing that :godo:

i figured out why it was doing that :godo:

**Review by whateverusername0 — APPROVED** [review]

File: Resources/Prototypes/_Goobstation/Catalog/Fills/Lockers/heads.yml [hunk 1]
```diff
@@ -37,6 +37,8 @@
     - id: PinpointerNuclear
     - id: ClothingHandsGlovesCombat
     - id: WeaponEnergyRevolver
+    - id: ClothingOuterHardsuitBlueshield
+    - id: OxygenTankFilled
```
Comment (Piras314) [inline Resources/Prototypes/_Goobstation/Catalog/Fills/Lockers/heads.yml:41]: needs nitrogen as well

**Review by Piras314 — CHANGES_REQUESTED** [review]

**Pull request merged**